docs/self-managed/about-self-managed.md

```diff
@@ -7,29 +7,38 @@ title: "Camunda 8 Self-Managed"
 
 Camunda 8 Self-Managed is not Camunda 7. [Find Camunda 7 documentation here](https://docs.camunda.org).
 
-However, Optimize documentation is available for both Camunda 8 and Camunda 7, including deployment instructions available [here]($optimize$/self-managed/optimize-deployment/install-and-start). Look for `Camunda 7` badges to help you understand what content is available for what product.
```

Comment:
I discussed the use of the Camunda 7 references with @akeller a few weeks ago... In the effort of reducing the amount of C7 content in the SM docs (and potential confusion), this seems like an easy way to start.